src/main/java/org/spongepowered/api/util/Tristate.java

```diff
@@ -94,6 +126,13 @@ public static Tristate fromBoolean(boolean val) {
      */
     public abstract Tristate or(Tristate other);
 
+    /**
+     * NOTs this tristate.
```

Comment:
Maybe call it negate?

Reply:
Edited

Reply:
Maybe also change the method name accordingly?

Reply:
IMO `not` lines up better with `and`/`or` than `negate` does.